css/partials/_sidebar.scss

```diff
@@ -1,16 +1,14 @@
 .sidebar {
```

Comment:
These changes are for restyling the sidebar widgets - removing the beige bg and using the top grey line to mark widgets.